### DISCUSSION TITLE
Does copilot still accept new users?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#8938] [Copilot Conversations] [answered]
So far I've been on the waitlist for ~4 months and was wondering if this whole acceptance thing is still available or did Github get the amount of people they needed and no longer accept new users?

I can wait if they are still accepting, im just really curious.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there!

We are still accepting new users. Glad you're excited about GitHub Copilot, but please know GitHub Support, the Feedback Discussion, and the Community Forum are unable to expedite this process for you.

As GitHub Copilot utilizes our state-of-the-art AI hardware, we are offering it to a limited number of testers for free, during the technical preview. We want to use the preview to learn how people use GitHub Copilot and what it takes to operate it at scale. If the technical preview is successful, our plan is to build a commercial version of GitHub Copilot in the future. As a result, the team is cautiously adding users to the technical preview as the product is refined. **_Users are not added on a first come first serve basis_** as we want to ensure a diverse group of users is testing the program. I understand how that can be frustrating, but this is in the hopes that one day this will be available to all users publicly so we really appreciate everyone’s patience.

For those waiting, please keep an eye out for an email if you are selected for the technical preview; it will be sent to the primary email address on your GitHub account.
